server/resources/ats.json

```diff
@@ -6,9 +6,8 @@
         "defaultConfigurationInstructionsHTML": "Configure JAWS with default settings. For help, read &lt;a href=&quot;https://github.com/w3c/aria-at/wiki/Configuring-Screen-Readers-for-Testing&quot;&gt;Configuring Screen Readers for Testing&lt;/a&gt;.",
         "assertionTokens": {
             "screenReader": "JAWS",
-            "interactionMode": "PC cursor active",
             "readingMode": "virtual cursor active",
-            "readingCursor": "virtual cursor"
+            "interactionMode": "PC cursor active"
```

Comment:
Can you revert the changes made to this file? Or please re-run `yarn db-import-tests:dev;` to make sure you have the latest version of this file.